### PR TITLE
CxPilot-7.0.0 Release

### PR DESCRIPTION
**Pull request by robertlong13**

### CODE REVIEW

File: ArduPlane/ReleaseNotes.txt [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+ - Roll limit no longer confined to 7 degrees during transition
+ - FBWB/CRUISE fix massive undershoot issue when switching from max descent to max climb
+ - A whole bunch of bug fixes
+
+Ottano Changes:
+ - Fixed-wing tune improvements. Altitude should be more stable.
+ - Battery assignments standardized
+   1. VTOL batteries combined (reports lowest voltage)
+   2. Avionics battery
+   3. Fuel level sensor
```
Comment (LachlanConn) [inline ArduPlane/ReleaseNotes.txt:21]: Maybe note that there is currently no failsafe in place for fuel level until we have a reliable reading on the fuel level sensor.

Comment (robertlong13) [reply ArduPlane/ReleaseNotes.txt:21]: I thought about adding that note, but there's never been a fuel level failsafe has there? If there never has been, I don't want it in the changelog.

**Review by loki077 — APPROVED** [review]
LGTM

File: ArduPlane/ReleaseNotes.txt (hunk 1 above)
Comment (Jono453) [reply ArduPlane/ReleaseNotes.txt:21]: How was the fuel level readings from O10 at marulan? Same new calibration will be in O11

**Review by Pradeep-Carbonix — APPROVED** [review]

**Pull request merged**